src/server/pps/cmds/cmds.go

```diff
@@ -298,6 +298,57 @@ Examples:
 		}),
 	}
 
+	var (
+		jobId       string
+		commaInputs string // comma-separated list of input files of interest
+	)
+	getLogs := &cobra.Command{
+		Use:   "get-logs",
+		Short: "Return logs from a job.",
+		Long:  "Return logs from a job.",
```

Comment:
Can we have examples of how this command is supposed to be used, similar to other commands?